Makefile

```
@@ -120,3 +120,7 @@ swagger-validate:
 		yelp/openapi-generator-cli:20201026 \
 		validate \
 		-i paasta_tools/api/api_docs/swagger.json
+
+.PHONY: my_life_easier
+my_life_easier:
```

as funny as this is, this should probably be named something that doesn't require reading the makefile to figure out what the target does
e.g.,
```suggestion
.PHONY: vscode_settings
vscode_settings:
```